ports/servoshell/Cargo.toml

```diff
@@ -52,34 +52,43 @@ xr-profile = ["libservo/xr-profile"]
 [dependencies]
 # For optional feature servo_allocator/use-system-allocator
 servo_allocator = { path = "../../components/allocator" }
+libc = { workspace = true }
+libservo = { path = "../../components/servo" }
+cfg-if = { workspace = true }
+log = { workspace = true }
+url = { workspace = true }
+lazy_static = { workspace = true }
+
 
 [target.'cfg(not(target_os = "android"))'.dependencies]
-arboard = { version = "3" }
 backtrace = { workspace = true }
-cfg-if = { workspace = true }
+# Todo: Could argument parsing be useful on non-desktop?
+getopts = { workspace = true }
```

Comment:
I'm not really sure if this makes sense, or if the whole argument parsing should just be desktop specific. 
Probably for android / ohos, we would just make a special `about:prefs` url, that allows editing an on-disk settings file?

Reply:
I removed the todo. I agree that its probably useful to keep it for now. We can always make it an optional dependency later (for non-desktop), if e.g. binary size becomes an important metric.